docs/quickstart/web.md

```diff
@@ -0,0 +1,53 @@
+---
+bodyclass: docs
+layout: docs
+headline: Web Quickstart
+sidenav: doc-side-quickstart-nav.html
+type: markdown
+---
+<p class="lead">This guide gets you started with gRPC-Web with a simple
+working example from the browser.</p>
+
+<div id="toc"></div>
+
+## Prerequisites
+
+* `docker` and `docker-compose`
+
+Please refer to [Docker website][] on how to install Docker.
+
+## Run an Echo example from the browser!
+
+```sh
+$ git clone https://github.com/grpc/grpc-web
+$ cd grpc-web
+$ docker-compose pull prereqs common echo-server envoy commonjs-client
```

Comment:
I was thinking about how to make this more explainable. 
Could we have a small subseciton to talk about the dependencies and what they do?
Ie:
The docker-compose plans pulls the following components:
- prereqs: ...
- echo-server -- a standard gRPC server we want to expose over the internet
- envoy -- the proxy which will translate between grpc and grpc-web. gRPC-Web is implemented as a filter in Envoy.
- commonjs-client -- the web based gRPC client side library that will be used to access the service.

It will also be very useful to have a block diagram that explains how these components are connected.

Reply:
Fixed